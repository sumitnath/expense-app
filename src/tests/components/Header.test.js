import React from "react";
//import ReactShallowRender, { createRenderer } from 'react-test-renderer/shallow'
import { shallow } from "enzyme";
import Header from '../../components/Header'
import toJson from 'enzyme-to-json'

test('should render Header correctly ', () => {
    //     const renderer = new ReactShallowRender();
    //     renderer.render( < Header / > )
    //     expect(renderer.getRenderOutput()).toMatchSnapshot()
    //     console.log(renderer.getRenderOutput());
    const wrapper = shallow( < Header / > )
        // expect(wrapper.find('h1').length).toBe(1)
        //  expect(wrapper.find('h1').text()).toBe('Expensify')
    expect(toJson(wrapper)).toMatchSnapshot()
})