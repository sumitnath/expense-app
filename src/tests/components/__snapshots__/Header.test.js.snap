// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly  1`] = `
<header>
   
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
     Go Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
     Add Expensee   
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
     Get Help
  </NavLink>
</header>
`;
